main.py

```diff
@@ -46,9 +50,9 @@ async def main():
         print(r)
         character_list_str = []
         for character_card in r.card:
-            character_fullname = "{}-{}".format(character_card.name.replace(' ','_'), character_card.rarity)
-            character_card_filename = "card-{}.jpg".format(character_fullname)
-            character_card.card.convert('RGB').save(os.path.join(outputdir, character_card_filename))
+            character_fullname = "{}-{}".format(character_card.id.replace(' ','_'), character_card.rarity)
+            character_card_filename = "card-{}.png".format(character_fullname)
+            character_card.card.save(os.path.join(outputdir, character_card_filename))
```

Comment:
The saving of JPEG format is intended since this could bring some benefit for data consumption (take template 3 as an example):
- PNG: 4.1M
- JPG: 1.3M

Since PNG is an uncompressed format and JPG is a compressed format. In my opinion, the benefit here is significant but the loss is so obvious for regular sized screens😂.